docs/src/app/components/pages/components/grid-list.jsx

```diff
@@ -87,19 +87,19 @@ class GridListPage extends React.Component {
           },
           {
             name: 'actionPosition',
-            type: '"left"|"right"',
+            type: 'oneOf [left, right]',
```

Comment:
thats fine, can change that. how about `oneOfType`? i dont think quotes make sense there?